crates/sui-verifier/src/entry_points_verifier.rs

```diff
@@ -28,13 +28,13 @@ use crate::{format_signature_token, resolve_struct, verification_failure, INIT_F
 /// - The function must have the name specified by `INIT_FN_NAME`
 /// - The function must have `Visibility::Private`
 /// - The function can have at most two parameters:
-///   - mandatory &mut TxContext (see `is_tx_context`) in the last position
+///   - mandatory &mut TxContext or &TxContext (see `is_tx_context`) in the last position
 ///   - optional one-time witness type (see one_time_witness verifier pass) passed by value in the first
 ///   position
 ///
 /// For transaction entry points
 /// - The function must have `is_entry` true
-/// - The function must have at least one parameter: &mut TxContext (see `is_tx_context`)
+/// - The function may have an optional &mut TxContext or &TxContext (see `is_tx_context`) parameter
```

Comment:
From Issue description and this comment, it's not clear if we assume the input to an entry function can be any of `&mut TxContext or &TxContext` or we can even omit `TxContext` completely.

Reply:
Updated the PR description and will fix the comment as well.